dynamics/wind_model.py

```diff
@@ -0,0 +1,215 @@
+import sys
+import os.path
+import csv
+import pandas as pd
+import math
+import matplotlib.pyplot as plt
+from routemodel.data_retrieval.get_weather import get_weather
```

Comment:
Hey, code looks good. Running it locally breaks here for me, does this import work on yours?